Projects/Foundations/Schemas/Docs-Topics/O/Omega/Omega-One-Data-Mine/omega-one-data-mine-008-conservative-moving-average.json

```diff
@@ -0,0 +1,268 @@
+                    "updated": 1647334097738
+                }
+            ]
+        },
+        {
+            "style": "List",
+            "text": "Enter the values of your choice.",
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Введите значения по своему выбору.",
+                    "updated": 1643526325298
+                },
+                {
+                    "language": "DE",
+                    "text": "Geben Sie die Werte Ihrer Wahl ein.",
+                    "updated": 1647334108380
+                }
+            ]
+        },
+        {
+            "style": "Note",
+            "text": "Remember that you have to delete existing data for the indicator in the Data-Storage folder, and re-start the Data Tasks needed for the indicator in the Network node before you can see any changes.",
+            "updated": 1617954918128,
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Помните, что необходимо удалить существующие данные для индикатора в папке Data-Storage и заново запустить Data Tasks, необходимые для индикатора, в узле Network, прежде чем вы сможете увидеть какие-либо изменения.",
+                    "updated": 1643526331219
+                },
+                {
+                    "language": "DE",
+                    "text": "Denken Sie daran, dass Sie die vorhandenen Daten für den Indikator im Ordner Data-Storage löschen und die für den Indikator erforderlichen Data Tasks im Knoten Network neu starten müssen, bevor Sie die Änderungen sehen können.",
+                    "updated": 1647334121729
+                }
+            ]
+        },
+        {
+            "style": "Png",
+            "text": "PNGs/Foundations/Docs/indicators/Omega-One-conservativeMA-crossing-params.png",
+            "updated": 1617958956443
+        },
+        {
+            "style": "Subtitle",
+            "text": "Products, Properties & Examples",
+            "updated": 1617967853007,
+            "translations": [
+                {
+                    "language": "RU",
+                    "text": "Продукты, свойства и примеры",
+                    "updated": 1643526343832
+                },
+                {
+                    "language": "DE",
+                    "text": "Produkte, Eigenschaften und Beispiele",
+                    "updated": 1647334133143
+                }
+            ]
+        },
+        {
+            "style": "Table",
+            "text": "| Product Name | Product Variable | Properties |\n| Conservative Moving Average | cma | cma |",
+            "updated": 1617972838194,
+            "translations": [
+                {
+                    "language": "DE",
+                    "text": "| Produktname | Produktvariable | Eigenschaften |\n| Conservative Moving Average | cma | cma |\n",
+                    "updated": 1647334180089
```

Comment:
And this file?

Reply:
This is because the file was renamed or moved. I've been through a lot of these myself.